flutter_todo/lib/components/widgets.dart

```diff
@@ -0,0 +1,114 @@
+import 'package:flutter/material.dart';
```

Comment:
for my own flutter education, why did you put all the widget elements in a single file?

Reply:
They are too small widgets. I moved them here so that the main widgets to be clearer. These widgets here are more like helpful pieces than meaningful for the user widgets.